### ISSUE TITLE
Question on training label

### ISSUE THREAD

**Issue by DianCh** [gaopengcuhk/CLIP-Adapter#6]
Hi! Thank you for release this interesting work. Just to check my understanding: during few-shot training, is the label made by pseudo labeling with the _original_ CLIP feature and classification weights (before the adapter layers)? Does the few-shot training set contain ground truth lables?

**Comment by Aikoin** [comment]
> Hi! Thank you for release this interesting work. Just to check my understanding: during few-shot training, is the label made by pseudo labeling with the _original_ CLIP feature and classification weights (before the adapter layers)? Does the few-shot training set contain ground truth lables?你好！感谢您发布这部有趣的作品。只是为了检查我的理解：在few-shot训练期间，标签是用_原始_CLIP特征和分类权重（在适配器层之前）伪标签制作的吗？几次训练集是否包含地面实况标签？

I think this work is in a supervised manner and the few-shot training sets all contain GT labels ( by the way, your adaContrast is the first TTA paper I read lol I love your presentation
